### PR TITLE
Create and populate research section.

### LINKED ISSUE #233: Content Suggestion: Research Section
Add a research section with links to documents.

### LINKED ISSUE #234: Content Suggestion: Bitcoin and the Trust Problem: Is Bitcoin adoption accelerated by the abuse of trust?
1) Link to Content: https://karozagorus.keybase.pub/KaroZagorus-2020-Thesis.pdf
2) Briefly Describe Content: Thorough research about why Bitcoin matters, how society is affected by the abuse of trust, how society fixes it and itself by using Bitcoin and how people organize around the bitcoin ecosystem to prove it protections.
3) Suggested Section: Research within Learning

### PR DESCRIPTION
**Pull request by karozagorus**
Added a new section
Added 6102bitcoin's coinjoin research.
Added karozagorus's Bitcoin thesis
Added the FCA's cryptoasset consumer research

Closes #234 
Closes #233

### CODE REVIEW

**Comment by karozagorus** [comment]
These were here added
https://github.com/bitcoin-only/bitcoin-only/issues/233
https://github.com/bitcoin-only/bitcoin-only/issues/234

**Comment by karozagorus** [comment]
OOps, I messed up 6102's name accidentally in it lol.

File: index.html [hunk 1]
```diff
@@ -1537,6 +1537,33 @@ <h2 id="text110" class="style3">Websites</h2>
...
+						    <table>
+						      <thead>
+						        <tr>
+						          <th>Class</th>
+						          <th>Description</th>
+						        </tr>
+						      </thead>
+						      <tbody>
+						        <tr>
+						          <td><a href="https://github.com/6102bitcoin/CoinJoin-Research">Coinjoing Research</a></td>
```
Comment (pardus79) [inline index.html:1552]: "Coinjoing" needs spelling correction.

Comment (karozagorus) [reply index.html]: Fixed! Wow github is pretty fucking cool, didn't know it would auto update it.

**Pull request merged**